FinniversKit/Sources/Fullscreen/EmptyView/GenericErrorSwiftUIView.swift

```diff
@@ -1,6 +1,12 @@
 import SwiftUI
 
-public final class GenericErrorViewModel {
+public final class GenericErrorViewModel: Equatable {
+    public static func == (lhs: GenericErrorViewModel, rhs: GenericErrorViewModel) -> Bool {
```

Comment:
I think It might be a bit nicer to move it to a separate extension below the class, but maybe I am just nitpicking here 😅

Reply:
Make sense, updated 👍